drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -298,12 +299,16 @@ function DragAndDropBlock(runtime, element, configuration) {
                     input.class_name = item_user_state.correct_input ? 'correct' : 'incorrect';
                 }
             }
+            var content_html = item.displayName;
+            if (item.backgroundImage) {
+                content_html = '<img src="' + item.backgroundImage + '" alt="' + item.backgroundDescription + '" />';
+            }
             var itemProperties = {
                 value: item.id,
                 drag_disabled: Boolean(item_user_state || state.finished),
                 class_name: item_user_state && ('input' in item_user_state || item_user_state.correct_input) ? 'fade': undefined,
                 input: input,
-                content_html: item.backgroundImage ? '<img src="' + item.backgroundImage + '"/>' : item.displayName
+                content_html: content_html
```

Comment:
Nit: We are only supporting ES5+ browsers, so you can add a trailing comma to objects/dicts like this if you'd like nicer diffs in the future, just like in Python :)